source/administration/migrating.rst

```diff
@@ -26,4 +26,70 @@ The following instructions migrate Mattermost from one server to another by back
 Migrating from Slack
 ====================
 
-You can import channels and users from Slack into Mattermost from Team Settings > Import. Please review our documentation on `Slack Import <https://docs.mattermost.com/help/settings/team-settings.html#import-from-slack-beta>`_ for more details.
+.. note:: As a proprietary SaaS service, Slack is able to change its export format quickly and without notice. If you encounter issues not mentioned in the documentation below, please alert the product team by `filing an issue <https://github.com/mattermost/platform/issues>`_.
+
+The Slack Import feature in Mattermost is in "Beta" and focused on supporting migration of teams of less than 100 registered users.
+
+This feature can be accessed through the `Mattermost Web App </administration/migrating.html#migrating-from-slack-using-the-mattermost-web-app>`_ or using the `CLI </administration/migrating.html#migrating-from-slack-using-the-mattermost-cli>`_.
+
+.. warning:: **It is highly recommended that you test Slack import before applying it to an instance intended for production.**
+
+   If you use Docker, you can spin up a test instance in one line (``docker run --name mattermost-dev -d --publish 8065:80 mattermost/platform``). If you don't use Docker, there are `step-by-step instructions <http://docs.mattermost.com/install/docker-local-machine.html>`_ to install Mattermost in preview mode in less than 5 minutes.
+
+Supported Features
+++++++++++++++++++
+
+The following key features are supported out of the box when importing from Slack:
+
+* User accounts with an email address set.
+
+* Public channels and the text messages posted in them by users.
+
+* Channel topic and purpose.
+
+* Automatically adding imported users to their channels.
+
+Messages with file attachments are imported as a message containing a link to Slack's servers by default. The file attachments themselves can be imported to Mattermost by using the `Slack Advanced Exporter <https://github.com/grundleborg/slack-advanced-exporter>`_ tool to add them to your archive before importing it.
+
+Bot and Integration messages are imported by default, but if you would like them to display with the appropriate username when imported, you should ensure that `Enable Integrations to Override Usernames </administration/config-settings.html#enable-integrations-to-override-usernames>`_ is set **before** doing the import.
+
+.. note:: Slack user accounts with the same email address as existing accounts on your Mattermost server will be merged into those accounts on import.
+
+Limitations
++++++++++++
+
+The following limitations are present when importing from Slack:
+
+* The Markdown subset used by Slack's Posts 2.0 feature announced on September 28, 2015 is not yet supported.
```

Comment:
Non-documentation question: Wondering if we could create a Help Wanted issue for it?

Someone might be interested in adding support for it.